### PR TITLE
Convert APP to typescript

### PR DESCRIPTION
**Pull request by rvsia**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -9,20 +11,30 @@ import { notificationsReducer } from '@redhat-cloud-services/frontend-components
...
     if (chrome) {
       const registry = getRegistry();
-      registry.register({ notifications: notificationsReducer });
+      registry.register({ notifications: notificationsReducer as Reducer });
       const { identifyApp, on: onChromeEvent } = chrome.init();
 
       // You can use directly the name of your app
       identifyApp(pckg.insights.appname);
-      unregister = onChromeEvent('APP_NAVIGATION', (event) =>
+      unregister = onChromeEvent('APP_NAVIGATION', (event: NavDOMEvent) =>
```
Comment (Hyperkid123) [inline src/App.tsx:37]: I am thinking about how we can make the type to be inferred. Gimme a bit of time.

Comment (rvsia) [reply src/App.tsx:37]: 🍹 🌴

Comment (Hyperkid123) [reply src/App.tsx:37]: @rvsia Can you try with these changes: https://github.com/RedHatInsights/frontend-components/pull/1568. Just copy paste the new types in node_modules

Comment (rvsia) [reply src/App.tsx]: Fixed

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**